src/glog/logging.h.in

```diff
@@ -1721,17 +1741,17 @@ class GOOGLE_GLOG_DLL_DECL LogSink {
   // during this call.
   virtual void send(LogSeverity severity, const char* full_filename,
```

Comment:
@sergiud , we can make this as pure virtual method and get rid of the below "send" method